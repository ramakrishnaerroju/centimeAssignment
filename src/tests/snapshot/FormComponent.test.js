import React from "react";
import renderer from "react-test-renderer";
import Form from "../../components/form/FormComponent";

test("Form snapshot test", () => {
  const mockCallBack = jest.fn();
  const component = renderer.create(<Form onSubmitCallback={mockCallBack} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
