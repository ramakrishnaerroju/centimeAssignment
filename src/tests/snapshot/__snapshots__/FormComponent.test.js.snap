// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form snapshot test 1`] = `
<div
  className="container"
>
  <h2>
    addData
  </h2>
  <form
    action="#"
    className="form-group"
    onReset={[Function]}
    onSubmit={[Function]}
  >
    <div>
      <label
        htmlFor="formControlInput1"
      >
        income
      </label>
      <input
        className="form-control"
        id="formControlInput1"
        name="income"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <br />
    <div>
      <label
        htmlFor="formControlInput2"
      >
        expenditure
      </label>
      <input
        className="form-control"
        id="formControlInput2"
        name="expenditure"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
    <div />
    <br />
    <div>
      <label
        htmlFor="formControlInput3"
      >
        amount
      </label>
      <input
        className="form-control"
        id="formControlInput3"
        name="amount"
        onBlur={[Function]}
        onChange={[Function]}
        type="number"
        value=""
      />
    </div>
    <div />
    <br />
    <button
      className="btn btn-primary"
      disabled={false}
      type="submit"
    >
      submit
    </button>
  </form>
</div>
`;
